src/lib.rs

```diff
@@ -25,7 +24,14 @@ mod fallback_hash;
 #[cfg(test)]
 mod hash_quality_test;
 
+#[cfg(feature = "compile-time-rng")]
 use const_random::const_random;
+
+#[cfg(not(feature = "compile-time-rng"))]
+macro_rules! const_random {
```

Comment:
There is no longer a need to define the macro here. On line 54 you can make it say:
```
#[cfg(not(feature = "compile-time-rng"))]
static SEED: AtomicUsize = AtomicUsize::new(MULTIPLE as usize);
```

Reply:
Done.